src/snyk/snykCode/codeService.ts

```diff
@@ -125,6 +125,11 @@ export class SnykCodeService extends AnalysisStatusProvider implements ISnykCode
           fileOptions: {
             paths,
           },
+          analysisContext: {
+            flow: this.config.source,
```

Comment:
What is the difference to `source` that is passed in the `connection` property above?

Reply:
from this perspective, nothing much
from our perspective (in octopus) they're used differently and might contain different values from consumers.

Reply:
Thanks for clarifying 👍